docs/workflow/requirements/windows-requirements.md

```diff
@@ -34,7 +34,7 @@ Visual Studio 2019 installation process:
 
 A `.vsconfig` file is included in the root of the dotnet/runtime repository that includes all components needed to build the dotnet/runtime repository. You can [import `.vsconfig` in your Visual Studio installer](https://docs.microsoft.com/en-us/visualstudio/install/import-export-installation-configurations?view=vs-2019#import-a-configuration) to install all necessary components.
 
-The dotnet/runtime repository requires at least Visual Studio 2019 16.3.
+The dotnet/runtime repository requires at least Visual Studio 2019 16.6 Preview 2.
```

Comment:
Is this version publicly available yet?

Reply:
Yes: https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview.

Reply:
Cool.